cmake/modules/CMakeCPackOptions.cmake.in

```diff
@@ -2,6 +2,35 @@
 # To pass variables to cpack from cmake, they must be configured
 # in this file.
 
+set(CPACK_PACKAGE_NAME "ROOT")
+set(CPACK_PACKAGE_VENDOR "CERN")
+set(CPACK_PACKAGE_DESCRIPTION "A modular scientific software toolkit")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://root.cern")
+set(CPACK_PACKAGE_CONTACT "ROOT Developers <root-dev@cern.ch>")
+
+if(CPACK_GENERATOR STREQUAL "DEB")
+  set(CPACK_DEBIAN_FILE_NAME "DEB-DEFAULT")
+  set(CPACK_DEBIAN_PACKAGE_SECTION "science")
+  set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS TRUE)
+  set(CPACK_DEBIAN_PACKAGE_GENERATE_SHLIBS TRUE)
+  set(CPACK_DEBIAN_PACKAGE_GENERATE_SHLIBS_POLICY "=")
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "g++")
+  set(CPACK_DEBIAN_PACKAGE_RECOMMENDS "python3, python3-numpy")
+  set(CPACK_DEBIAN_PACKAGE_SUGGESTS "cmake, make, jupyter, python3-numba")
```

Comment:
should we add all packages listed at https://root.cern/install/dependencies/#ubuntu-and-other-debian-based-distributions ?

Reply:
Up to ROOT team to decide later, I added what I thought made sense at the time. You can either add or remove as you wish. However, I think at least make and cmake should stay, as most likely people will want those to use ROOT with their own code. You could create different variants of ROOT packages, for example, with few or many dependencies by changing what's installed at the time you build ROOT and the package, and/or changing this list.

Reply:
I think we should offer deb/rpm packages with support for as many features as possible: I am sure 99% of users would rather pay a bit more disk space than figure out which of several packages they need. 

But independently of that, I don't understand why just make and cmake are enough, shouldn't we list at least ROOT's required packages here, lest users end up with a broken installation?

For comparison, the arch package has the following dependencies:

```
Depends On      : blas  desktop-file-utils  fcgi  fftw  ftgl  giflib  gl2ps  glew  graphviz  gsl  hicolor-icon-theme  intel-tbb  libafterimage  librsvg  libxpm  python python-numpy  tex-gyre-fonts  unixodbc  xxhash>=0.6.5-1  zstd
Optional Deps   : cern-vdt: Add a set of fast and vectorisable mathematical functions
                  chromium: Support for WebGUI
                  cfitsio: Read images and data from FITS files
                  libmariadbclient: MySQL support
                  libxml2: XML parser interface
                  openmp: Support OpenMP extensions in Minuit2
                  openmpi: Support OpenMPI extensions in Minuit2
                  openssl: OpenSSL support
                  postgresql-libs: PostgreSQL support
                  pythia8>=8.2.40-1: Pythia8 EG support
                  qt5-webengine: Support for WebGUI
                  sqlite: SQLite support
                  tcsh: Legacy CSH support
                  unuran: Support non-uniform random numbers
                  vc: Add types for portable and intuitive SIMD programming
                  xrootd4: Support remote file server and client
                  z3: Suuport the Z3 theorem prover
                  gcc-fortran: Enable the Fortran components of ROOT
```

Reply:
Most dependencies are figured out by CPack automatically, only the ones it misses have to be listed here.